common/docserver/docserver.go

```diff
@@ -95,14 +95,20 @@ func (self *DocServer) Get(uri, path string) (content []byte, err error) {
 			resp.Body.Close()
 		}
 	}()
+
 	if err != nil {
 		return
 	}
+
 	content, err = ioutil.ReadAll(resp.Body)
 	if err != nil {
 		return
 	}
 
+	if resp.StatusCode >= 400 {
```

Comment:
Yes they could be returned. Is there an other canonical way to match errors. I could certainly add 304 but wonder if that is the only one.